content/en/docs/ops/configuration/multi-revision/gateways/index.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Configuring and managing gateways with multiple revisions
+description: Configuring and managing gateways with multiple revisions.
+weight: 25
+keywords: [kubernetes,upgrading]
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+With a single `IstioOperator` CR, any gateways defined in the CR (including the `istio-ingressgateway` installed in the
+default profile) are upgraded in place, even when the canary control plane method is used. This is
+undesirable - gateways are a critical component that should be upgraded last, when the new version control and data
+planes are verified to be working.
+The recommended way to upgrade gateways is to define and manage them in a separate `IstioOperator` CR, separate from
+the one used to install and manage the control plane.
+
+{{< warning >}}
+To avoid problems with `.` (dot) not being a valid character in some Kubernetes paths, the revision name should not
+include `.` (dots).
+{{< /warning >}}
+
+## Istioctl
+
+This section covers installation and upgrade of separate control plane and gateways, using `istioctl`.
+
+### Installation with `istioctl`
+
+1.  Ensure that the main `IstioOperator` CR has a name and does not install a gateway:
+
+{{< text yaml >}}
+# filename: control-plane.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: `IstioOperator`
+metadata:
+  name: control-plane # REQUIRED
+spec:
+  profile: default
+  components:
+    ingressGateways:
+      - name: istio-ingressgateway
+        # MUST BE DISABLED
+        enabled: false
+{{< /text >}}
```

Comment:
Here and throughout the document, any text that is associated with a numbered list item must be indented 4 spaces. Otherwise the next step (on line 45) will reset to 1 in the generated page (see preview https://deploy-preview-8299--preliminary-istio.netlify.app/latest/docs/ops/configuration/multi-revision/gateways/#installation-with-istioctl).

Reply:
done